src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -730,9 +567,8 @@ int32_t NaClSysOpen(struct NaClAppThread  *natp,
     memmove(left_side, right_side, len_of_rest);
   }
 
-  if (retval) {
-    return retval;
-  }
+  if (retval) return retval;
+  
```

Comment:
Why this change?

Reply:
just cosmetic, was trying to standardize the look of these functions